playwright.config.ts

```diff
@@ -0,0 +1,103 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+    testDir: './e2e',
+    /* Maximum time one test can run for. */
+    timeout: 30 * 1000,
+    expect: {
+        /**
+         * Maximum time expect() should wait for the condition to be met.
+         * For example in `await expect(locator).toHaveText();`
+         */
+        timeout: 5000,
+    },
+    /* Run tests in files in parallel */
```

Comment:
Does this mean that the test() calls in one file are run in parallel or that tests in different files are run in paralllel?

Reply:
All tests are run in parallel, assuming there are enough workers.

Reply:
(However, all workers/contexts have their own state, including calling beforeAll once per worker, so no account is ever used in parallel from multiple tests)